### DISCUSSION TITLE
Run development branch along side latest in docker

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2118] [Help] [answered]
Hi is there a way I can run the development branch of BB apps alongside the latest bb apps version from [here ](https://hub.docker.com/r/budibase/apps/tags?page=1&ordering=last_updated). there are quite a few bugs I can not workaround that I'm told are fixed in this branch. How would I go about running the development branch? would I need to install a completely different back end or will the two bb/apps run alongside each other?

**Comment by shogunpurple** [comment, marked as answer]
@AndrewJRamay there is not. We will be performing a release tomorrow with all the latest bug fixes and features. Thank you for your patience.

**Reply by KernOil** [reply]
thank you for the quick reply, that is great news.  keep up your awesome work!
